### PR TITLE
Template 11 Project page

### PR DESCRIPTION
**Pull request by neelesh-tippe**
[UND-95][story/UND-95-PROJECT-PAGE-TEMPALTE11]: Implement project page

### CODE REVIEW

File: stories/assets/js/Carousel.js [hunk 1]
```diff
@@ -2,47 +2,47 @@ import arrowright from '../images/Icon/arrow-right.svg';
 import arrowleft from '../images/Icon/arrow-left.svg';
 
 export function CarouselFun(ele, gapele, gapele_sm, viewcard) {
-
   /**
-   * A Custom component to get Glide Slides Length and Width.
+   * A Custom Length component to get Glide Slides Length.
```
Comment (shishirphadke) [inline stories/assets/js/Carousel.js:6]: @neelesh-tippe Kindly verify the file as per develop branch. Please don't revert the changes from develop.

File: stories/Templates/ProjectPage/projectpage.scss [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+          @include devicebreak(medium) {
+            font-size: 3rem;
+          }
+        }
+      }
+    }
+  }
+}
+
+/* ***RTL styling end ****/
```
Comment (shishirphadke) [inline stories/Templates/ProjectPage/projectpage.scss:198]: Remove the comment here.

File: stories/Templates/ProjectPage/projectpage.scss [hunk 3]
```diff
@@ -0,0 +1,198 @@
+@import '../../assets/scss/mixins';
+@import '../../assets/scss/breakpoints';
+@import '../../assets/scss/variables';
+
+/* ***Project Page style  *** */
```
Comment (shishirphadke) [inline stories/Templates/ProjectPage/projectpage.scss:5]: Remove the comment here.

File: stories/Templates/ProjectPage/projectpage.scss [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+      margin-top: 3.75rem;
+
+      @include devicebreak(medium) {
+        margin-top: 4.375rem;
+      }
+    }
+  }
+}
+
+/* ***Project Page style end ****/
```
Comment (shishirphadke) [inline stories/Templates/ProjectPage/projectpage.scss:144]: Remove the comment here.

File: stories/Templates/ProjectPage/projectpage.scss [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+
+/* ***RTL styling ****/
```
Comment (shishirphadke) [inline stories/Templates/ProjectPage/projectpage.scss:146]: Remove the comment here.

File: stories/Components/Forms/Newslettersignup/SignUp.stories.mdx [hunk 6]
```diff
@@ -47,5 +47,5 @@ export const getCaptionForLocale = (locale) => {
     return (
       <SignUp element='input' type='email' required='required' mode='form-email' errorText={caption.errorLabel} label={caption.labelname} button={caption.buttonname} placeholder={caption.placeholdername} {...args}></SignUp>
     );
-  }}
+  }}
```
[the remark this answers is not in the document]
Comment (neelesh-tippe) [reply stories/Components/Forms/Newslettersignup/SignUp.stories.mdx]: Done changes

File: stories/assets/js/Carousel.js (hunk 1 above)
Comment (neelesh-tippe) [reply stories/assets/js/Carousel.js]: done

File: stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx [hunk 7]
```diff
@@ -62,7 +62,7 @@ export const ContentCardWithAndWithoutImage = ({
                 <></>
             )}         
         </div>
-      ))}
+      ))}
```
Comment (shishirphadke) [inline stories/Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage.jsx:65]: @neelesh-tippe I still see whitespace here. Kindly remove.

File: stories/Organism/Statscardslider/Statscardslider.jsx [hunk 8]
```diff
@@ -1,6 +1,6 @@
-import React, { Component, useEffect } from 'react';
+import React, { Component, useEffect } from "react";
```
Comment (shishirphadke) [inline stories/Organism/Statscardslider/Statscardslider.jsx:1]: @neelesh-tippe Kindly use single quotes `'` here.

File: stories/Organism/Statscardslider/Statscardslider.jsx [hunk 9]
```diff
@@ -1,6 +1,6 @@
...
 import { CarouselFun } from '../../assets/js/Carousel';
-import { StatsCards } from '../../Components/UIcomponents/Cards/StatsCards/StatsCards';
+import { StatsCards } from "../../Components/UIcomponents/Cards/StatsCards/StatsCards";
```
Comment (shishirphadke) [inline stories/Organism/Statscardslider/Statscardslider.jsx:3]: @neelesh-tippe Kindly use single quotes `'` here.

File: stories/Templates/CountryHomepage/CountryHomepage.jsx [hunk 10]
```diff
@@ -104,6 +106,9 @@ export const CountryHomepage = ({
               data={featureddataafterwedo.contentdata }
               storyBtn={featureddataafterwedo.storyBtn }
             />
+            <div className="cell cta-container">
+                <CtaButton label={storyBtn} For_Primary='No Arrow' />
```
Comment (shishirphadke) [inline stories/Templates/CountryHomepage/CountryHomepage.jsx:110]: @neelesh-tippe Change the text here. Avoid using underscore for inline attributes `For_Primary`.

File: stories/Templates/ProjectPage/ProjectPage.jsx [hunk 11]
```diff
@@ -0,0 +1,177 @@
...
+import { Footer } from "../../Organism/Footer/Footer";
+import { Heading } from "../../Atom/Typography/Heading/Heading";
+import { P } from '../../Atom/Base-typography/Paragraph/Paragraph';
+import { DownloadCard } from '../../Molecules/Blocks/DownloadCard/DownloadCard';
+import { Projectdetail } from '../../Organism/Text/ProjectDetail/ProjectDetail';
+import { CtaButton } from '../../Components/UIcomponents/Buttons/Cta_button/CtaButton';
+import { ContentCardWithAndWithoutImage } from '../../Organism/Blocks/ContentCardWithAndWithoutImage/ContentCardWithAndWithoutImage';
+import { Icondownload } from '../../Atom/Icons/download';
+import { Statscardslider } from '../../Organism/Statscardslider/Statscardslider';
+import "./projectpage.scss";
```
Comment (shishirphadke) [inline stories/Templates/ProjectPage/ProjectPage.jsx:15]: @neelesh-tippe Kindly use single quotes `'` in the import statements. You can read more about here:
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: stories/Templates/CountryHomepage/CountryHomepage.jsx (hunk 10 above)
Comment (neelesh-tippe) [reply stories/Templates/CountryHomepage/CountryHomepage.jsx]: CTA button is imported. This change need to be do on base component.
